regression/cpp/Template_Specialisation2/main.ii

```diff
@@ -15,3 +15,6 @@ struct __member_pointer_traits_imp<_Rp (_Class::*)() const>
 {
 };
 
+int main(int argc, char* argv[])
```

Comment:
Should this become `main.cpp` as with the other file ( @tautschnig )

Reply:
I don't know -- there isn't really enough information in the initial commit message for that file (e155e42030921), so I'd suggest to leave it as is. It should not have any impact. Should.